samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/GeneralMessages_ca.properties

```diff
@@ -23,7 +23,6 @@ t_eventLog=Ves a la p\u00e0gina de registre d'esdeveniments
 t_section_activity=V\u00e9s a la p\u00e0gina amb l'informe d'activitat de l'usuari
 t_restore_assessments=Ves a la paperera
 # Accesskey: all accesskey name start with a_
-a_log=l
```

Comment:
What happens in the JSF template where this is referenced?
samigo-app/src/webapp/jsf/evaluation/evaluationHeadings.jsp

Reply:
it falls back to the default GeneralMessages.properties

Reply:
I'm probably being stupid here, but if a_log=l is still the property, how will this be changed the behaviour described in the  JIRA ticket.

Reply:
look at the default property file GeneralMessages.properties

Reply:
Ah, right. You're putting g in there. got it.